docs/debugging-s2i.md

```diff
@@ -0,0 +1,116 @@
+
+Troubleshooting
+=================
+
+This document contains some tips and suggestions for troubleshooting the use of source-to-image ("s2i") to inject source into Docker images.
+
+Can't Find The `sti` Executable
+--------------
+
+Whether by  running `make build`,  running `hack/build-release.sh`, or downloading one of our official releases,  look under the `_output/local/go/bin` directory.
```

Comment:
`make build` and `build-release.sh` will output to given directory, but downloading official release not, btw. add a link to that release page, it's more user friendly :wink: